src/bin/wasmer.rs

```diff
@@ -247,6 +248,11 @@ struct Run {
     #[structopt(long = "enable-experimental-io-devices", hidden = true)]
     enable_experimental_io_devices: bool,
 
+    /// Enable debug output
```

Comment:
Is there a canonical naming for the logging flags? I would expect that `--debug` would map precisely to enabling `debug!()` and above. Something like `--log-level=...` which corresponds to the enum in `Log::LevelFilter`, defaulting to `Off` given that we conditionally initialize logging?

Or maybe we should set it to `Error` or `Warn` and always initialize logging? I like that even more.

Reply:
Yeah, so currently I'm using Cargo features for this!